installer/cmd/tectonic/main.go

```diff
@@ -16,8 +16,9 @@ var (
 	clusterInstallCommand = kingpin.Command("install", "Create a new Tectonic cluster")
 	clusterInstallDirFlag = clusterInstallCommand.Flag("dir", "Cluster directory").Default(".").ExistingDir()
 
-	clusterDestroyCommand = kingpin.Command("destroy", "Destroy an existing Tectonic cluster")
-	clusterDestroyDirFlag = clusterDestroyCommand.Flag("dir", "Cluster directory").Default(".").ExistingDir()
+	clusterDestroyCommand   = kingpin.Command("destroy", "Destroy an existing Tectonic cluster")
+	clusterDestroyDirFlag   = clusterDestroyCommand.Flag("dir", "Cluster directory").Default(".").ExistingDir()
+	clusterDestroyContOnErr = clusterDestroyCommand.Flag("continue-on-error", "Log errors, but attempt to continue cleaning up the cluster.  THIS MAY LEAK RESOURCES, because you may not have enough state left after a partial removal to be able to perform a second destroy.").Default("false").Bool()
```

Comment:
nit: Is there a precedent for `--continue-on-error`?  `--keep-going` is [in Make][1].  Consistency with existing tools isn't critical, but it's nice to reuse existing wording where we can.

[1]: https://www.gnu.org/software/make/manual/html_node/Options-Summary.html#index-_002d_002dkeep_002dgoing-2

Reply:
I was thinking of [spf13/pflag](https://github.com/spf13/pflag/blob/master/flag.go#L120) which is, admittedly, obscure.